app/controllers/api/lists_controller.rb

```diff
@@ -0,0 +1,21 @@
+class Api::ListsController < ApiController
+  before_action :authenticated?
+  
+  def index
```

Comment:
this action needs to render json including all lists to be rendered. Because the action is empty, it uses Rails default rendering and tries to render a view with the name of the action.